packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -171,4 +261,56 @@ test.describe('PieInput - Component tests', () => {
             });
         });
     });
+
+    test.describe('Form integration', () => {
+        test('should appear within the form data object when the form is submitted', async ({ page }) => {
```

Comment:
I think this test description needs updating slightly. Something like below:

`should set the "formDataObject" value of username to the correctly when submitted`